src/gallia/db/handler.py

```diff
@@ -27,7 +27,7 @@ def bytes_repr(data: bytes) -> str:
     return bytes_repr_(data, False, None)
 
 
-schema_version = "1.0"
+schema_version = "1.1"
```

Comment:
Let's use schema `2`. Major/Minor schemas do not make sense IMO.

Reply:
fixed